Dockerfile

```diff
@@ -18,6 +18,9 @@ ARG COMPOSER_NO_DEV=1
 # This is NOT compatible with the release target above
 ARG BRANCH=master
 
+# https://stackoverflow.com/questions/53377176/change-imagemagick-policy-on-a-dockerfile
+ARG IMAGEMAGIC_CONFIG=/etc/ImageMagick-6/policy.xml

```

Comment:
I don't see why we need this?

Reply:
[6b17554](https://github.com/LycheeOrg/Lychee-Docker/pull/225/commits/6b1755400f1d43ab72236d5b58e8dd581f937f5f)